src/go/configmanager/flags/flags.go

```diff
@@ -60,6 +60,7 @@ var (
 
 	SslServerCertPath                = flag.String("ssl_server_cert_path", "", "Path to the certificate and key that ESPv2 uses to act as a HTTPS server")
 	SslServerCipherSuites            = flag.String("ssl_server_cipher_suites", "", "Cipher suites to use for downstream connections as a comma-separated list.")
+	SslServerRootCertsPath           = flag.String("ssl_server_root_cert_path", "", "The file path of root certificates that ESPv2 uses to verify downstream client certificate. If not specified, ESPv2 doesn't the validation by default.")
```

Comment:
ESPv2 doens't verify client certificate.

Reply:
Done.

Reply:
I still see this "ESPv2 doesn't the validation by default".  This is not right grammar